farm_scripts/ddsim.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+source /opt/detector/epic-nightly/setup.sh
+
+tempdir=/scratch/$USER/$JOBNAME
+mkdir -p ${tempdir}
+
+datadir=${tempdir}/data
+logdir=${tempdir}/logs
+outdir=${tempdir}/rootfiles
+
+mkdir -p ${datadir}
+mkdir -p ${logdir}
+mkdir -p ${outdir}
+
+cp $WORK_FILE $datadir
+
+datafile=$datadir/$BASEFILE
+outfile=$outdir/$BASENAME.edm4hep.root
+logfile=$logdir/$BASENAME.log
+
+export JUGGLER_MC_FILE=$datafile
+export JUGGLER_SIM_FILE=$outfile
+    
+nohup ddsim --steeringFile steering.py \
```

Comment:
Why does this need a nohup, is this not dangerous on the farm?

Reply:
Im not sure. I've always kept nohup in these type of scripts to have compatibility when not using the farm. Never ran into issues on the farm but i can investigate

Reply:
removed for safety anyway.